regression/utils.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utility and helper functions for regression testing."""
+
+import os
+import tempfile
+from typing import Optional
+
+_CLOUD_REMOTE_LOCATIONS = {
+    'gs': 'gs://mosaicml-composer-tests/streaming/regression/',
```

Comment:
Is it possible to fetch the path during runtime ? I mean user would need to provide a remote_dir to run! We would like avoid putting any cloud location publicly.

Reply:
Hmm wouldn't it still go in the yaml then? And can be seen publicly anyways? Also it seems like we expose our cloud location elsewhere
https://github.com/mosaicml/streaming/blob/137928c595d503ae3c996ff0ea1980a91f949318/tests/test_streaming_remote.py#L24